### PR TITLE
Refactor PasswordPrompt

### PR DESCRIPTION
**Pull request by pivilartisant**
old structure :
```
├── main.tsx
├── pages
│   ├── ImportPrivateKey
│   │   ├── PromptNickname.tsx
│   │   └── PromptPrivateKey.tsx
│   ├── backupKeyPairs.tsx
│   ├── backupMethods.tsx
│   ├── confirmDelete.tsx
│   ├── failure.tsx
│   ├── importFile.tsx
│   ├── importMethods.tsx
│   ├── newPassword.tsx
│   ├── passwordPrompt.tsx
│   └── success.tsx
```

new structure :
```
├── main.tsx
├── pages
│   ├── ImportPrivateKey
│   │   ├── PromptNickname.tsx
│   │   └── PromptPrivateKey.tsx
│   ├── PasswordPrompt
│   │   ├── Default.tsx
│   │   ├── Delete.tsx
│   │   ├── Sign.tsx
│   │   ├── Transfer.tsx
│   │   └── passwordPromptHandler.tsx
│   ├── backupKeyPairs.tsx
│   ├── backupMethods.tsx
│   ├── confirmDelete.tsx
│   ├── failure.tsx
│   ├── importFile.tsx
│   ├── importMethods.tsx
│   ├── newPassword.tsx
│   └── success.tsx
```

### CODE REVIEW

**Review by Thykof — CHANGES_REQUESTED** [review]
my partial review

File: wails-frontend/src/pages/PasswordPrompt/Default.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    if (!Success) {
+      if (CodeMessage === ErrorCode.WrongPassword) {
+        setError({ password: Intl.t(`errors.${CodeMessage}`) });
+        return;
+      }
+      handleApplyResult(navigate, req, setError, false)(result);
+    } else {
+      handleApplyResult(navigate, req, setError, false)(result);
+    }
```
[the remark this answers is not in the document]
Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPrompt/Default.tsx]: let me know if pushed changes are what you expected

**Review by mazmassa — APPROVED** [review]
Thanks for that Paul! 👏

**Comment by pivilartisant** [comment]
Double checking flow here, pls wait b4 merge !!

**Comment by Thykof** [comment]
we ignore sonar lint CI

**Pull request merged**